build.gradle

```diff
@@ -181,8 +181,31 @@ subprojects {
       exceptionFormat = 'full'
     }
 
+    useJUnit {
+      excludeCategories 'org.apache.kafka.test.IntegrationTest'
+    }
+
+  }
+
+  task integrationTest(type: Test, dependsOn: compileJava) {
+    maxParallelForks = userMaxForks ?: Runtime.runtime.availableProcessors()
+
+    minHeapSize = "256m"
+    maxHeapSize = "2048m"
+    jvmArgs = maxPermSizeArgs
+
+    testLogging {
+      events = userTestLoggingEvents ?: ["passed", "skipped", "failed"]
+      showStandardStreams = userShowStandardStreams ?: false
+      exceptionFormat = 'full'
+    }
+    useJUnit {
+      includeCategories 'org.apache.kafka.test.IntegrationTest'
+    }
   }
 
+  task testAll(dependsOn: ['test', 'integrationTest'])
```

Comment:
`testAll` currently runs all the tests for all Scala versions. So looks like this clashes with that. Could we have unitTest, integrationTest, test (unit and integration for the default Scala version) and testAll (unit and integration for all Scala versions)?